lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -588,12 +589,52 @@ def get_snapshot_status(response: Dict[str, Any] | None) -> BackupServiceState:
         return BackupServiceState.SUCCESS
 
 
+class _DisableBackupRelationEvent(EventBase):
```

Comment:
Interesting. Is this to avoid defers/returns on the `relation_broken` event?

Reply:
Yes, to defer the relation_broken safely.